### PR TITLE
role and policy and deployement

### PR DESCRIPTION
**Pull request by szuecs**
This will enable users to use static egress for defined targets in configmap with label egress=static

### CODE REVIEW

File: cluster/manifests/kube-static-egress-controller/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
```
Comment (mikkeloscar) [inline cluster/manifests/kube-static-egress-controller/deployment.yaml]: apps/v1beta1 :)

Comment (szuecs) [reply cluster/manifests/kube-static-egress-controller/deployment.yaml]: yeah, sure :)

File: cluster/manifests/kube-static-egress-controller/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  template:
+    metadata:
+      labels:
+        application: kube-static-egress-controller
+        version: v0.0.3
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+        iam.amazonaws.com/role: "{{ .LocalID }}-static-egress-controller"
+    spec:
+      serviceAccountName: system
```
Comment (mikkeloscar) [inline cluster/manifests/kube-static-egress-controller/deployment.yaml]: The controller only needs to read from the API right? In that case I would drop this and just use the default service account which has readOnly access.

**Comment by szuecs** [comment]
:+1:

**Comment by linki** [comment]
👍

**Pull request merged**